README.md

```diff
@@ -18,11 +18,24 @@ $ ./gradlew build
 ```
 
 ### Configuration
-Services to authenticate are retrieved from environment variables in the following format:
+
+In order to setup Service Auth Provider to work with a client service, you need to do the following:
+
+* In the Azure Key Vault named `s2s-{environment}` add the service's secret used for generating OTPs (one-time passwords).
+This has to be done in each environment the service is going to be deployed to. Service Auth Provider will use that secret
+for validating OTPs. It has to be a BASE32-encoded sequence of ten random bytes (16 characters after encoding). By convention,
```

Comment:
it would be worth either adding a oneliner here that will generate a secret in an appropriate form  or a script to this repo, if its quick could you do it now? or a separate PR?

Reply:
Good idea. Done.